helm/cert-manager-app/values.schema.json

```diff
@@ -16,6 +16,9 @@
                                     "properties": {
                                         "AcmeSolver": {
                                             "type": "string"
+                                        },
+                                        "DNSServer": {
+                                            "type": ["null", "string"]
```

Comment:
From what I remember, an empty string is a null value as far as helm is concerned

Reply:
`null` as value is a special thing in Helm templates. It means: Delete that key/value pair from the manifest. So maybe you want allow that for every single value, else you can't use that feature. On the other hand in most cases it won't be used. One use case is `minAvailable` and `maxUnavailable` for [pdb](https://kubernetes.io/docs/tasks/run-application/configure-pdb/#specifying-a-poddisruptionbudget). If in the default values one is set, the user can't just overwrite by setting the other in his config. They need to `null` the other one. So here `null` could make sense for `DNSServer` in case by default `AcmeSolver` would be `dns01` and the user wants to overwrite with `http01` and delete `DNSServer`. Setting to `""` might have the same effect depending on the app.

Reply:
I see. All good then.